src/main/java/scrublords/misc/BossMechanics.java

```diff
@@ -0,0 +1,19 @@
+package scrublords.misc;
+
+import scrublords.main.GamePanel;
+import scrublords.entities.characters.Player;
+/**
+ * @author Nikolay Zahariev <nikolay.g.zahariev@gmail.com>.
+ */
+public class BossMechanics {
+
+    public void levelOneBossBattleStartTeleport(Player player) {
```

Comment:
This method teleports the player to the boss, the name could be changed to something better. If it's only going to be used for bosses you can call it **teleportToBoss** if it can be reused you can call it just **teleportPlayer** if you don't like these names think of something else.